R/quotation.R

```diff
@@ -1,109 +1,95 @@
-#' Quotation
+#' Defusing R expressions
 #'
 #' @description
 #'
 #' \Sexpr[results=rd, stage=render]{rlang:::lifecycle("stable")}
 #'
-#' Quotation is a mechanism by which an expression supplied as
-#' argument is captured by a function. Instead of seeing the value of
-#' the argument, the function sees the recipe (the R code) to make
-#' that value. This is possible because R [expressions][is_expr] are
-#' representable as regular objects in R:
+#' When a function argument is defused, R doesn't return its value
+#' like it normally would. Instead, it returns the R expression
+#' describing how to make the value as an inert object. These defused
+#' expressions are like blueprints for computing values.
 #'
-#' * Calls represent the action of calling a function to
-#'   compute a new value. Evaluating a call causes that value to be
-#'   computed. Calls typically involve symbols to reference R objects.
+#' The main purpose of preventing evaluation of an expression is to
```

Comment:
Is that really the main purpose?

Reply:
Data masking isn't the main purpose?

Reply:
I think you need to rewrite the sentence to be more like: "The main purpose of defusing evaluation to allow data-masking, where an expression is evaluated in the context of a data frame so that you can do `var` instead of `df$var`.